File: Plus_minus.py
def decimal_count(numbers):
    plus=0
    minus=0
    zeroes=0
    for i in numbers:
        if i>0:
            plus+=1
        elif i<0:
            minus+=1
        else:
            zeroes+=1
    positives=plus/len(numbers)
    negatives=minus/len(numbers)
    zero=zeroes/len(numbers)

    print(f"Positive decimal values is :{positives}")
    print(f"Negative decimal values is :{negatives}")
    print(f"Zeroes decimal values is :{zero}")
    
numbers=[-1,5,-9,-8,-2,0,0,0,4,3,8]
decimal_count(numbers)

File: Plus_minus.java
package Arrays;
import java.util.*;
import java.io.*;
public class Plus_minus{
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int[] arr= new int[n];
        for (int i = 0; i < arr.length; i++) {
            arr[i]=sc.nextInt();
        }
        for (int i = 0; i < arr.length; i++) {
            System.out.print(arr[i]+" ");
        }
        int plus=0;
        int minus=0;
        int zero=0;
        for (int i = 0; i < arr.length; i++) {
            if(arr[i]>0){
                plus++;
            }
            else if(arr[i]<0) {
                minus++;
            }
            else{
                zero++;
            }
        }
        float positives=plus/n;
        float negatives=minus/n;
        float zeros=zero/n;
        System.out.printf("Positive fraction is :",positives);
        System.out.printf("Negative decimals is :",negatives);
        System.out.printf("Zeroes decimal is :",zeros);
    }
}